QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+QUEST_JOBSTEP_20150317_000781	$According to the anecdote, Rumpelstiltskin has not reached the moon yet, and is still flying there with the magic in his body. {nl}It could also be the case that he returned before his demonic life span ended.{nl}
+QUEST_JOBSTEP_20150317_000782	$Nevertheless, as we know, Lydia Schaffen has passed away long ago. So while the demon did not lose the game, he has lost his chance to win the game forever. {nl}Still, the one matter left that we, as the current generation, are curious about is what the agreed compensation between Lydia Schaffen and Rumpelstiltskin was. {nl}Of course, we could ask a particular person, err, there is a thing that may still exist. However, as we do not know whether he has returned or not, it will be hard to settle this lingering matter for quite some time.{nl}{nl}
+QUEST_JOBSTEP_20150317_000783	$Atman Extension
+QUEST_JOBSTEP_20150317_000784	$Atman Extension
+QUEST_JOBSTEP_20150317_000785	$Congratulations on receiving this book, which no criminal may see. There are many ways to use magic, but the magic used by Linkers are a particular case. Other mages exercise effect-based powers rather than magic. That is to say, to fight an enemy with a fireball, or with lightning, does not truly bind the enemy with magic. 
+QUEST_JOBSTEP_20150317_000786	$These attacks only use magic to make a fireball and lightning. Then, the mages controls the magical energy's direction and destination, releasing it after settling on both.
+QUEST_JOBSTEP_20150317_000787	$In other words, these mages are unable to handle enemies without Linkers; as they need to channel and regulate surrounding energy, these mages do not have the time to easily handle the targeting their magic towards an enemy. However, as you have chosen to walk the path of the Linker, the way of your magic will differ completely. The way of a Linker's magic is by directly linking your magic with your opponent's magic, which includes searching for your target within seconds.
+QUEST_JOBSTEP_20150317_000788	$Also, in the case of linking more than 2 targets, a Linker's way of magic is complicated even further if the targets have mutually opposing properties. In these situations, the Linker must use a viable magical property to connect the targets with opposing properties.
+QUEST_JOBSTEP_20150317_000789	$Thus to some extent, a Linker are experienced magic users who still have a lot of new fields to test. Even if there are those predestined to be Linkers, fledging mages who have developed their abilities to control magic are recommended becoming Linkers and walking the path of a Linker. Above all, the most important matter about Linkers is the establishment of their truly resolved soul, or Atman. Without it, it is possible for attacks directed at a magically linked target to return towards the Linker.
```

Comment:
"a Linker are experienced magic users" -> "a Linker is an experienced magic user" or "the Linkers are experienced magic users"